src/data/zmod/basic.lean

```diff
@@ -274,60 +274,98 @@ begin
   refl
 end
 
-variable [char_p R n]
+section char_dvd
+/- If the characteristic of `R` divides `n`, then `cast` is a homomorphism. -/
```

Comment:
```suggestion
/-! If the characteristic of `R` divides `n`, then `cast` is a homomorphism. -/
```
Unless they're about implementation details, I think comments like these can be promoted to module docs.